nautobot/docs/administration/nautobot-server.md

```diff
@@ -545,6 +552,29 @@ Note that there is presently no option to provide input parameters (`data`) for
 
 Please see the [guide on Jobs](../additional-features/jobs.md) for more information on working with and running jobs.
 
+### `send_installation_metrics`
+
++++ 1.6.0
+
+`nautobot-server send_installation_metrics`
+
+Send installation metrics to the Nautobot maintainers. This management command is called by [`post_upgrade`](#post_upgrade) and is not intended to be run manually.
+
+If the [`INSTALLATION_METRICS_ENABLED`](../configuration/optional-settings.md#installation_metrics_enabled) setting is `True`, this command will send a list of all installed [plugins](../plugins/index.md) and their versions, as well as the currently installed Nautobot version to the Nautobot maintainers. A randomized UUID will be generated and saved in the [`DEPLOYMENT_ID`](../configuration/optional-settings.md#deployment_id) setting to anonymously but uniquely identify this installation. The plugin names will be one-way hashed with SHA256 to further anonymize the data sent.
```

Comment:
Maybe should copy-paste the wording from version-1.6.md to here, looks like there's some drift currently.